### PR TITLE
[SCB-1823] change kie label env to environment

### PR DESCRIPTION
**Pull request by GuoYL123**

### CODE REVIEW

File: dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ public static String encrypt(String dataStr) {
       //todo:how to deal env
       if (labelsMap.containsKey("app") && labelsMap.get("app")
           .equals(KieConfig.INSTANCE.getAppName())
-          && labelsMap.containsKey("env") && labelsMap.get("env")
+          && labelsMap.containsKey("environment") && labelsMap.get("environment")
```
Comment (liubao68) [inline dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java]: Too many literals in a function, better to define constants for them.

File: dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java [hunk 2]
```diff
@@ -65,7 +65,7 @@ public static String encrypt(String dataStr) {
       //todo:how to deal env
```
Comment (liubao68) [inline dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java]: Can this todo removed?

File: dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java (hunk 1 above)
Comment (GuoYL123) [reply dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java]: done

File: dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java (hunk 2 above)
Comment (GuoYL123) [reply dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieUtil.java]: done

**Pull request merged**